Khan/wonder-blocks
# Add jsx addon

This addon allows developers to see the JSX source code for all of our
stories.  This makes storybook a valuable documentation tool for developers.
    
Test Plan:
    - npm run storybook
    - load http://localhost:9001
    - open the "jsx" tab in the addon panel
    - see the JSX for the story
    
Reviewers: john idr miguel
